common/beerocks/bcl/source/beerocks_logging.cpp

```diff
@@ -440,10 +448,11 @@ void logging::handle_logging_rollover(const char *log_name, std::size_t)
     remove(rollover_name.c_str());
     rename(log_name, rollover_name.c_str());
 }
+
 void logging::apply_settings()
 {
     // Disable The instance of RollMonitor to start fresh
-    {
+    if (m_log_files_auto_roll) {
```

Comment:
Should probably be a separate commit which adds the auto-roll option

Reply:
Could've split the changes into more atomic commits, but since most of the changes are concentrated in just a couple of files (mostly `beerocks_logging.cpp`), I thought it'll be easier to review by seeing the bigger picture.
Will take that into account for the next PRs.